Composer/packages/server/src/models/bot/process/orchestratorWorker.ts

```diff
@@ -179,14 +189,10 @@ const handleMessage = async (msg: RequestMsg) => {
       }
     }
   } catch (error) {
-    return { id: msg.id, error };
+    process.send?.({ id: msg.id, error: { message: error?.message, stack: error?.stack } });
```

Comment:
shouldn't error be empty if there are no error?
```error: error ? { message: error.message, stack: error.stack } : undefined```

Reply:
Thanks, addressed